CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing Guidelines
+
+🎉 **First of all, thanks for taking the time to contribute!** 🎉
+
+The following is a set of guidelines for contributing to this project. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+
+## How can I contribute? 🤔
+
+### Reporting Bugs 💣
+
+This section guides you through submitting a bug report. Following these guidelines helps maintainers and the community understand your report 📝, reproduce the behavior 💻 💻, and find related reports 🔎
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+### Suggesting Enhancements 🛠
+
+This section guides you through submitting an enhancement suggestion, including completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion 📝 and find related suggestions 🔎.
```

Comment:
I think you should remove a mention of features as this repo isn't for a software project. I don't think we will be having anything we can call a feature.